runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentVideoViewClient.java

```diff
@@ -29,14 +29,7 @@ public XWalkContentVideoViewClient(XWalkContentsClient client, Activity activity
     @Override
     public void enterFullscreenVideo(View view) {
         mView.setOverlayVideoMode(true);
-        CustomViewCallback cb = new CustomViewCallback() {
-            @Override
-            public void onCustomViewHidden() {
-                ContentVideoView contentVideoView = ContentVideoView.getContentVideoView();
-                if (contentVideoView != null) contentVideoView.exitFullscreen(false);
```

Comment:
Yes, please look at https://github.com/crosswalk-project/chromium-crosswalk/blob/master/content/browser/web_contents/web_contents_impl.cc#L1486, exit fullscreen will be requested from render frame host.